src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+
+public class DeadlineTest {
+    private String invalidDate = "40/40/999";
+    private String validDate = "1/1/2022";
+    private String validDescription = "test description";
+    /**
+     * Test Deadline with valid input
+     */
+    @Test
+    public void test1() {
```

Comment:
I think you can consider renaming the method to be more descriptive. Perhaps in whatIsBeingTested_descriptionOfTestInputs_expectedOutcome format?